src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -213,6 +229,7 @@ function positionNodeLabel(year) {
 
 function grabCursor(event) {
     yearCursorIsGrabbed = true
+    isPristine.value = false
```

Comment:
I would wait for the value to be altered before setting pristine to false, so that if the user grab but then releases the cursor without changing anything (miss-click) it doesn't block its settings with the preview year